pkg/kubectl/resource_printer.go

```diff
@@ -334,9 +358,10 @@ type PrintOptions struct {
 // will only be printed if the object type changes. This makes it useful for printing items
 // received from watches.
 type HumanReadablePrinter struct {
-	handlerMap map[reflect.Type]*handlerEntry
-	options    PrintOptions
-	lastType   reflect.Type
+	handlerMap    map[reflect.Type]*handlerEntry
+	options       PrintOptions
+	lastType      reflect.Type
+	hiddenPodsNum int
```

Comment:
Added changes for reuse the same var `hiddenObjNum` for all objects and valid printing in case of user getting several resource types at once: kubectl get pods,namespaces (added output example in PR description)